packages/access-client/package.json

```diff
@@ -24,12 +25,12 @@
     "rc": "npm version prerelease --preid rc"
   },
   "exports": {
-    ".": "./src/index.js",
-    "./agent": "./src/agent.js",
-    "./drivers/*": "./src/drivers/*.js",
-    "./stores/*": "./src/stores/*.js",
-    "./types": "./src/types.js",
-    "./encoding": "./src/encoding.js"
+    ".": "./dist/src/index.js",
```

Comment:
changing where this points to resolved some atty errors. When we do it this way and let typescript also write the type declaration files with all its defaults next to the source files, then when another tsc in a depending lib finds all this config, it'll be able to find the src,types,etc again. I learned some of this from https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseESM.md#common-causes

Reply:
Can you update the npm published files so we don't include src in the root?

Reply:
@alanshaw https://github.com/web3-storage/w3up/pull/1012